<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Secure Shell Interface</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border: 1px solid;
	border-radius: 5px;
	padding: 10px 20px;
	background-color: white;
}
QLabel{
	font-weight: bold;
	color: white;
}
QPushButton:hover {
	background-color: grey;
}
QLineEdit{
	border: 1px solid;
	border-radius: 5px;
	padding: 5px 10px;
	background-color: white;
}
QLineEdit:hover{
	background-color: grey;
}
QMenuBar{
	color: white;
	border: 1px solid white;
}
QMenu{
	color: white;
	border: 1px solid white;
}
QScrollArea {
	border: 1px solid white;
}
* {
	background-color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="folder_area">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>321</width>
      <height>381</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>319</width>
       <height>379</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="file_area">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>321</width>
      <height>381</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>319</width>
       <height>379</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="folders_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Folders</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="files_label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Files</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionDisconnect"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_Connection">
   <property name="text">
    <string>Save Connection</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionCommand_Prompt">
   <property name="text">
    <string>Command Prompt</string>
   </property>
  </action>
  <action name="actionAdd_File">
   <property name="text">
    <string>Add File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
